src/scripts/towns/ProfOakNPC.ts

```diff
@@ -0,0 +1,41 @@
+class ProfOakNPC extends NPC {
+
+    constructor(
+        public name: string,
+        public dialog: string[]
+    ) {
+        super(name, dialog);
+    }
+
+    get dialogHTML(): string {
+        const UniquePokemonCaught = App.game.party.caughtPokemon.filter(p => p.id > 0).length;
+
+        if (UniquePokemonCaught >= 493) {
+            return `<p>Congratulations, you're more than half-way completed on the national Pokédex!</p>
+                    <p>Next stop is Unova! I've always wanted to visit Castelia City personally...</p>`;
+        }
```

Comment:
Could we also add a default message that will appear for any newer regions, otherwise this just adds another place that will need to be updated whenever a new region is added.